### PR TITLE
- Fixes minor spelling- and grammar mistakes

### PR DESCRIPTION
**Pull request by danielkhan**
- Changes wording for more conciseness
- Changes wording to include platform- and hardware vendors

### CODE REVIEW

**Comment by danielkhan** [comment]
Revalidate

File: spec/01-abstract.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
 ## Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology used by tracing tools to follow, analyze and debug a trace across multiple software components. Typically, a trace traverses more than one component which requires it to be uniquely identifiable across all participating systems. Passing along this unique identification is referred to as *context propagation*.
```
Comment (mtwo) [inline spec/01-abstract.md]: > analyze and debug a trace

Should still be "analyze and debug a transaction" or "analyze or debug a request". I'd recommend leaving the original paragraph.

File: spec/01-abstract.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
 ## Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology used by tracing tools to follow, analyze and debug a trace across multiple software components. Typically, a trace traverses more than one component which requires it to be uniquely identifiable across all participating systems. Passing along this unique identification is referred to as *context propagation*.
 
-Today context propagation is implemented in proprietary formats individually by each tracing tool provider. This results in siginficant interoperability problems between tracing tools than manifest in two major ways:
+Today, context propagation is implemented individually by each vendor.
```
Comment (mtwo) [inline spec/01-abstract.md]: I prefer the original lines 5 and 6

File: spec/01-abstract.md [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
+In the past, these problems did not have a significant impact as most applications were monitored by a single tracing tool and stayed within the boundaries of a single platform provider. Today, an increasing number of applications are highly distributed and leverage multiple middleware services and cloud platforms.
 
 ## Solution
 
-The trace context specification defines a universally agreed format for the exchange of context propagation data - referred to as trace context. Trace context solved the problems described above by
+The trace context specification defines a universally agreed-upon format for the exchange of context propagation data - referred to as *trace context*. Trace context solves the problems described above by
 
-- providing a unique identifier for individual tracing allowing traces of multiple providers being linked together and 
-- providing an agreed-upon mechanism to forward vendor-specific trace data avoid broken tracing when multiple trace tools particpate in a single transaction. 
+- providing a unique identifier for individual tracing, allowing traces of multiple providers to be linked together and
```
Comment (mtwo) [inline spec/01-abstract.md]: > individual tracing

should be "traces" I think, though this was incorrect in the original text as well

**Review by mtwo — CHANGES_REQUESTED** [review]

File: spec/01-abstract.md (hunk 2 above)
Comment (danielkhan) [reply spec/01-abstract.md]: I added another reason - so wording should be "[...] in three major ways"?
Is 'major ways' grammatically correct?

File: spec/01-abstract.md (hunk 1 above)
Comment (danielkhan) [reply spec/01-abstract.md]: I changed the wording to transaction.

File: .gitignore [hunk 4]
```diff
@@ -20,4 +20,7 @@ bin
 \#*\#
 
 # Vim
-.swp
\ No newline at end of file
+.swp
+
+#VSCode
+.vscode
```
Comment (yurishkuro) [inline .gitignore:26]: this goes to `~/.gitignore_global`

File: spec/01-abstract.md [hunk 5]
```diff
@@ -1,19 +1,39 @@
-## Problem Statement
+# Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology implemented by tracing tools to follow, analyze
+and debug a transaction across multiple software components. Typically, a trace
+traverses more than one component which requires it to be uniquely identifiable
+across all participating systems. Passing along this unique identification is
+referred to as *context propagation*.
```
Comment (yurishkuro) [inline spec/01-abstract.md]: it's a _job_ of context propagation, it's not _the_ context propagation - the latter can exist and be useful without unique identifier.

File: spec/01-abstract.md (hunk 2 above)
Comment (yurishkuro) [reply spec/01-abstract.md]: s/context propagation/trace context propagation/

File: spec/01-abstract.md [hunk 6]
```diff
@@ -1,19 +1,39 @@
...
 
-Today context propagation is implemented in proprietary formats individually by each tracing tool provider. This results in siginficant interoperability problems between tracing tools than manifest in two major ways:
+Today, context propagation is implemented individually by each vendor.
+In multi-vendor environments, this causes interoperability problems, like:
 
-- Traces that are collected by two tracing providers cannot be linked together as there is no unique identifier linking the individual pieces together. 
-- Traces are broken each time they are passed between components monitored by different tracing as there is no uniformly agree set of identifcation taht is forwarded.
+- Traces that are collected by different tracing vendors cannot be linked together
+- as there is no shared unique identifier.
```
Comment (yurishkuro) [inline spec/01-abstract.md]: this is not supposed to be a separate bullet, is it?

**Comment by yurishkuro** [comment]
Meta comment: this project is not about context propagation, only **trace context** propagation.

Cf. https://cs.brown.edu/~jcmace/papers/mace18universal.pdf

File: spec/01-abstract.md (hunk 6 above)
Comment (danielkhan) [reply spec/01-abstract.md]: Fixed in https://github.com/w3c/trace-context/pull/224/commits/951a7c920882acefb88bd0286d5549df9c3382c6

File: .gitignore (hunk 4 above)
Comment (danielkhan) [reply .gitignore:26]: Please explain

File: spec/01-abstract.md (hunk 2 above)
Comment (danielkhan) [reply spec/01-abstract.md]: @yurishkuro Changed in https://github.com/w3c/trace-context/pull/224/commits/738650b3c37799d62f3abbfa9315d0792435f46e
Likely, this needs to be changed throughout the spec then.

File: spec/01-abstract.md (hunk 5 above)
Comment (danielkhan) [reply spec/01-abstract.md]: Fixed with #244

**Comment by danielkhan** [comment]
@mtwo please review the requested changes

**Review by mtwo — APPROVED** [review]
LGTM, thanks Daniel!

**Pull request merged**